src/Explorer/QueryCopilot/QueryCopilotTab.tsx

```diff
@@ -191,7 +199,8 @@ export const QueryCopilotTab: React.FC<QueryCopilotTabProps> = ({ explorer }: Qu
       };
       setShowDeletePopup(false);
       useQueryCopilot.getState().refreshCorrelationId();
-      const response = await fetch("https://copilotorchestrater.azurewebsites.net/generateSQLQuery", {
+      const serverInfo = useNotebook.getState().notebookServerInfo;
+      const response = await fetch(`${serverInfo.notebookServerEndpoint}generateSQLQuery`, {
```

Comment:
Use createUri() from src\Common\UrlUtility to create the url instead of concatenating.